cloudinary-core/src/main/java/com/cloudinary/metadata/AbstractMetadataField.java

```diff
@@ -0,0 +1,113 @@
+package com.cloudinary.metadata;
```

Comment:
why not under com.cloudinary.api.metadata?

Reply:
`Api` itself is not under the api package, I tried to be consistent...